### ISSUE TITLE
CDx: Subscribe to CDx events

### ISSUE THREAD

**Issue by nditada** [instedd/hub#104]
Using the CDx API subscribe to a saved filter notifications when a new task is created in the hub for that saved filter.

**Comment by bcardiff** [comment]
resolved at c5e3246

* reference counting was implemented in order to subscribe only once per event.
* subscribe/unsubscribe was implemented when needed. (ie: `rc: 0 -> 1` /  `rc: 1 -> 0`)
* subcriber_id is stored in order for this to happen.
* event token based authentincation for the endpoint is used (check `callback/connectors/:id/*path` route and `api#callback` action)
* POST verb is used from CDX to receive JSON formatted event

**Comment by pmallol** [comment]
Ok in version 1a700c0
